packages/components/tests/dummy/app/styles/components/dummy-link-cta-button-banner.scss

```diff
@@ -0,0 +1,32 @@
+// notice: here we use custom typography to make the banner more compact
+
+.dummy-link-cta-button-banner {
```

Comment:
You'll need to add a different background color for the code block, it has to have a 3:1 contrast to the lavender background color. 

![CleanShot 2022-04-06 at 07 27 46](https://user-images.githubusercontent.com/4587451/161974724-7c6c8fab-b5bf-47e4-a791-eaf48f8e6dcf.png)

Reply:
OK, I'll leave the code without background then, it's easier.